runtime/doc/options.txt

```diff
@@ -6501,7 +6501,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	tags file are relative to the directory where the tags file is.
 
 						*'tags'* *'tag'* *E433*
-'tags' 'tag'		string	(default "./tags,tags", when compiled with
+'tags' 'tag'		string	(default "./tags;,tags", when compiled with
 				|+emacs_tags|: "./tags,./TAGS,tags,TAGS")
```

Comment:
That's something I was planning on correcting. It seems to be handled automatically, but I haven't checked where that is done yet. If you see it, let me know ;)